octobot/community/errors_upload/errors_uploader.py

```diff
@@ -63,12 +73,13 @@ def _ensure_upload_task(self):
             )
 
     async def _upload_errors(self, session, errors):
-        async with session.post(self.upload_url, json=self._get_formatted_errors(errors)) as resp:
-            if resp.status != 200:
-                self.logger.error(
-                    f"Impossible to upload error : status code: {resp.status}, text: {await resp.text()}",
-                    skip_post_callback=True
-                )
+        for error in errors:
+            with sentry_sdk.push_scope() as scope:
+                scope.set_tag("title", error.title)
+                scope.set_tag("type", error.type)
+                scope.set_user({"id": error.metrics_id})
+                if not sentry_sdk.capture_exception(error.error, scope=scope):
```

Comment:
is it async by the way ?

Reply:
It spawns a thread in background that sends captures. So this call should be non blocking.